.gitignore

```diff
@@ -163,4 +163,5 @@ dist
 
 # End of https://www.toptal.com/developers/gitignore/api/node,macos
 
-package-lock.json
\ No newline at end of file
+package-lock.json
+pnpm-lock.yaml
```

Comment:
The `package-lock.json` was listed here to prevent developers from committing the lockfile generated by `npm` (we wan't them to use pnpm instead). If we include the pnpm-generated lockfile we might end up with non-determinism across environments when we do `pnpm install` in the project.